mm2src/coins/utxo/rpc_clients.rs

```diff
@@ -175,7 +176,16 @@ impl UtxoRpcClientEnum {
                     },
                     Err(e) => {
                         if e.get_inner().is_tx_not_found_error() {
-                            return ERR!("Tx {} is not on chain anymore", tx_hash);
+                            error!(
+                                "Error {} getting tx from chain, retrying in 10 seconds. Retry left: {}",
```

Comment:
The error msg can be `Tx not found on chain error: {}, retrying in 10 seconds. Retries left: {}`